### PR TITLE
Renames Chemistry to Chemical Factory, changes Pharmacy back to Chemistry Lab

### PR DESCRIPTION
**Pull request by cacogen**
## About The Pull Request

Instead of keeping Chemistry the way it was and naming the chem factory Chemical Factory, some genius decided to rename Chemistry "Apothecary" and call the chemical factory "Chemistry". Eventually "Apothecary" became "Pharmacy". The change has lead to confusion over what Chemistry refers to, and boomers like myself resent Chemistry being renamed Pharmacy for no reason. 

In addition to this, the maps that don't use the chem factory haven't had their Chemistry Lab renamed to Pharmacy. Even some of the maps that do have the chem factory still have things in the Pharmacy that refer to it as Chemistry.

This PR removes the notion of a Pharmacy entirely, returning the Chemistry Lab and calling the chem factory what it is.

I also added some lights and cameras to the BoxStation chem factory while I was there as it was missing them after my xenobio PR expanded it downwards. I had to test the cameras there after renaming their tags and discovered blind spots and darkness.

## Why It's Good For The Game

- No more confusion over what Chemistry refers to
- Don't have to rename Chemistry on existing maps to Pharmacy if they have the chem factory added
- Get to keep calling the Chemistry Lab Chemistry

## Changelog
:cl: cacogen
tweak: Pharmacy is now the Chemistry Lab again. Chemistry (chem factory) is now called the Chemical Factory
fix: Incorrect/duplicate camera tags in BoxStation's Chemical Factory
add: Extra light and camera to BoxStation's Chemical Factory to fix some blind spots and dark spots
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
you'll get over it boomer

**Comment by cacogen** [comment]
This took ages to do and makes more sense than the current set up. Why wouldn't you want it renamed chemical factory at the very least?

**Comment by Buggy123** [comment]
24 hour rule, reopen this.

**Comment by TheVekter** [comment]
I'm that genius, I did it because it didn't make any fucking sense to have it called "Chemical Factory". I renamed it Pharmacy because the idea is that the chemistry lab that's in medical is supposed to (and I know this is a shocking revelation) _make medicine_. The 'chemical factory' is more for if you want to screw around making meth or big plumbing setups to generate shitloads of pills.

tl;dr I'm not arguing semantics, stop trying to overwrite my changes for no good reason.

**Comment by cacogen** [comment]
People find it confusing calling the chem factory chemistry when the pharmacy used to be called that. What about calling the room that produces large amounts of reagents using industrial grade automated machines a Chemical Factory doesn't make sense to you?

**Comment by TheVekter** [comment]
I can accept that, but there's literally no reason to rename the pharmacy to just Chemistry again aside from semantics.

**Comment by cacogen** [comment]
I did it because I'm a boomer and it's always been called that and I wanted it to stay that way and I didn't know it'd cause such a strong reaction. If the pull was still open I would've changed it back.

**Comment by ExcessiveUseOfCobblestone** [comment]
cringe

**Comment by cacogen** [comment]
?

**Comment by Time-Green** [comment]
I was the first genius to name it apothecary. Reason is it wouldn’t make any sense calling pharmacy chemistry, since it’s not solely for the chemists.

**Comment by ExcessiveUseOfCobblestone** [comment]
they still act like it does, I've seen some who don't even go to the factory and just treat it like old chem.

very sad!
